account/tests/urls_tests/test_client.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from .test_profile import TestProfileViews, FAKE_ID
+
+
+BASE_URL = '/client'
+
+
+@pytest.mark.django_db
+class TestClientViews:
```

Comment:
Whats does this class aims to do? 
Test view? test UI through client, 
I think you should organize things here